examples/edge_intelligence/Readme.md

```diff
@@ -0,0 +1,20 @@
+This folder contains example files for different models: pfa, sklearn, tensorflow, rule.
+
+# RuleEdgeComponent
+
+Rule edge component is like different edge components provided by Liota, you can define a simple rule depending on your requirement as a lambda function and the rule edge component will process the incoming metrics and give output to the actuator on the basis of the rule you have defined as lambda function.
```

Comment:
one of the edge intelligences components.

Lambda <Link Python lambda>

The result can be passed onto actuator on basis of the rule you have defined as lambda function.